Realm/RLMRealmConfiguration.h

```diff
@@ -60,6 +60,9 @@ RLM_ASSUME_NONNULL_BEGIN
 /// The block which migrates the Realm to the current version.
 @property (nonatomic, copy, nullable) RLMMigrationBlock migrationBlock;
 
+/// A custom schema to use for the Realm.
+@property (nonatomic, copy, nullable) RLMSchema *customSchema;
```

Comment:
RLMSchema exists, but the guide never needs to mention it and the sort of user that could be confused by it will never encounter it.